dashboard/test/controllers/api/v1/pd/workshops_controller_test.rb

```diff
@@ -103,6 +103,71 @@ class Api::V1::Pd::WorkshopsControllerTest < ::ActionController::TestCase
     assert_equal workshop_in_progress.id, response[0]['id']
   end
 
+  # Action: filter
+  test 'admins can filter' do
```

Comment:
It's included in the `all_forbidden` (see line 526 below). Do you want an explicit test case for this too?